<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LEDDialog</class>
 <widget class="QDialog" name="LEDDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LED</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/bulbs.png</normaloff>../../Downloads/bulbs.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>LED Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="2">
       <widget class="QCheckBox" name="allOnCheckBox">
        <property name="text">
         <string>ALL ON</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="ledFourCheckBox">
        <property name="text">
         <string>LED 4</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="ledThreeCheckBox">
        <property name="text">
         <string>LED 3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>43</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="ledTwoCheckBox">
        <property name="text">
         <string>LED 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="ledOneCheckBox">
        <property name="text">
         <string>LED 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="allOffCheckBox">
        <property name="text">
         <string>ALL OFF</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
